src/components/HeaderLoggedIn/index.js

```diff
@@ -46,6 +52,28 @@ export default class HeaderLoggedIn extends PureComponent {
     this.setState({ isMobileMenuOpened: !isMobileMenuOpened })
   }
 
+  renderLeft = () => {
+    const { left } = this.props
```

Comment:
move `type`, `action` and `label` to this object destructuring to avoid repeating `left`:

```javascript
const { type, action, label } = this.props.left
```